### DISCUSSION TITLE
Can't connect PhpStorm and CoPilot

### DISCUSSION THREAD

**Discussion by Yuseinov** [community/community#48150] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I can't connect `CoPilot` to `PhpStorm`.
In idea.log I have `INFO - #copilot - [INFO] [auth] [2023-02-21T13:17:21.541Z] Invalid copilot token: missing token: 403`.
Everything is on the latest version. 
I tried to uninstall and reinstall the plugin. 
In `/login/device/success` everything is OK: `Congratulations, you're all set!` `Congratulations, you're all set!` but it didn't connect and didn't work.

**Comment by iamcjwilliams** [comment]
Hello, @Yuseinov Have you visited [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot) to troubleshoot your issue?

**Comment by Yuseinov** [comment]
Yes. I visited here and from there I view the log info and post it in my issue as in the example in this link.
I fixed the problem. This error is not very descriptive. I fixed it when I added card details for payment to my account.
I can't even use the free subscription without card details and I can't understand the error. I found another problem of the same type without solutions. Just add some card details and everything will be fine.

**Comment by iamcjwilliams** [comment]
@Yuseinov Have you tried doing a clean uninstall and reinstall the extension?

**Reply by Yuseinov** [reply]
Sorry for my late response.
Yes, I tried many time uninstall, reinstall, restart, update, reading error logs, and etc.
Nothing fixed that, except adding a card to my account. 
Now, after two weeks, the extension works fine. I didn't have any problems.
